### PR TITLE
docs: A step-by-step guide to deploy on K8S

### PR DESCRIPTION
**Pull request by chncaesar**
## What is proposed in this PR?
A step-by-step guide to deploy on K8S

## How was this PR tested?
Test is not needed, but review is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,85 @@
 # MLSQL-Deploy
+mlsql-deploy provides a CLI to deploy MLSQL Engine on K8S. The process consists of
+- mlsql-deploy CLI build
```
Comment (allwefantasy) [inline README.md:3]: Build mlsql-deploy from source

File: README.md [hunk 2]
```diff
@@ -1,49 +1,85 @@
...
+- JucieFS and K8S setup
```
Comment (allwefantasy) [inline README.md:4]: Setup JuiceFS FileSystem

File: README.md [hunk 3]
```diff
@@ -1,49 +1,85 @@
...
+- Deploy with CLI on K8S
```
Comment (allwefantasy) [inline README.md:5]: Run mlsql-deploy CLI

File: README.md [hunk 4]
```diff
@@ -1,49 +1,85 @@
 # MLSQL-Deploy
+mlsql-deploy provides a CLI to deploy MLSQL Engine on K8S. The process consists of
```
Comment (allwefantasy) [inline README.md:2]: There are three steps:

File: README.md [hunk 5]
```diff
@@ -1,49 +1,85 @@
...
+- JucieFS and K8S setup
+- Deploy with CLI on K8S
 
 ## Build
-
 ```
 make all
 ```
 
 ## Command
```
Comment (allwefantasy) [inline README.md:12]: ## Test

File: README.md [hunk 6]
```diff
@@ -1,49 +1,85 @@
...
 ./mlsql-deploy --help
 ```
 
-## Deploy MLSQL Engine on K8s
+It's recommended to install JuiceFS to provide a distributes fileSystem. Together with K8S, it provides user a local
+cloud mimic environment.
 
-You need first downloading [JuiceFS 0.15.2](https://github.com/juicedata/juicefs/releases/tag/v0.15.2) then run following
-command:
+## [JuiceFS](https://github.com/juicedata/juicefs) Setup
```
Comment (allwefantasy) [inline README.md:21]: Change the tile

File: README.md [hunk 7]
```diff
@@ -1,49 +1,85 @@
...
+docker run -d --name minio \
+        -v ~/minio-data:/data \
+        -p 9000:9000 \
+        -p 9001:9001 \
+        --restart unless-stopped \
+        minio/minio server /data --console-address ":9001"
+ 
+```
+Login [MinIO cosnole](http://127.0.0.1:9001/) and create a bucket named jfs
+### Database Setup
```
Comment (allwefantasy) [inline README.md:37]: Meta Storage Setup

File: README.md [hunk 8]
```diff
@@ -1,49 +1,85 @@
...
+	--storage minio \
+    --bucket http://127.0.0.1:9000/jfs \
+    --access-key minioadmin \
+    --secret-key minioadmin \
+	redis://192.168.31.95:6379/1 \
+	jfs
+```
 
-Now you can deploy MLSQL Engine on k8s:
+## K8S Setup
```
Comment (allwefantasy) [inline README.md:62]: We can delete this part

**Review by allwefantasy — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (chncaesar) [reply README.md:12]: Add a "Test" topic? I'm not sure.

File: README.md (hunk 1 above)
Comment (chncaesar) [reply README.md]: Change is force-pushed, please click "latest change here" link

File: README.md (hunk 7 above)
Comment (chncaesar) [reply README.md]: updated.

**Review by allwefantasy — APPROVED** [review]

**Review by lwz9103 — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -1,49 +1,81 @@
...
 
 ## Command
-
 ```shell
 ./mlsql-deploy --version
 ./mlsql-deploy --help
 ```
 
-## Deploy MLSQL Engine on K8s
+It's recommended to install JuiceFS to provide a distributes fileSystem. Together with K8S, it provides user a local
```
Comment (lwz9103) [inline README.md:18]: file system?

**Review by allwefantasy — APPROVED** [review]

**Pull request merged**